nh_clinical/tests/res_user/__init__.py

```diff
@@ -1 +1 @@
-from . import test_check_location_ids_constraint
+# from . import test_check_location_ids_constraint
```

Comment:
Ah that was a mistake, I think I disabled it for a moment to see if the rest of the tests passed and then forgot to revert it. I've reinstated it not and moved it to the correct module so that it passes.